ui-frontend/packages/catalog-ui-search/src/main/webapp/component/layers/layers.view.tsx

```diff
@@ -97,7 +97,7 @@ const LayersViewReact = () => {
               }
             )
             user.get('user>preferences').get('mapLayers').sort()
-            // user.get('user>preferences').save()
+            user.get('user>preferences').savePreferences()
```

Comment:
Is this necessary since `saveCallback` is getting called when `mapLayers` changes? Also, it looks like this line was commented out before. If so, is the `savePreferences` in the reset to defaults onClick is also unnecessary?

Reply:
lol oops, I just read the PR description 🤦‍♀️  My bad!